bigframes/operations/semantics.py

```diff
@@ -50,8 +50,61 @@ def filter(self, instruction: str, model):
                 columns are referred to.
         """
 
+        output_instruction = "Based on the provided context, reply to the following claim by only True or False:"
+
+        from bigframes.dataframe import DataFrame
+
+        results = typing.cast(
+            DataFrame, model.predict(self._make_prompt(instruction, output_instruction))
+        )
+
+        return self._df[
+            results["ml_generate_text_llm_result"].str.lower().str.contains("true")
+        ]
+
+    def map(self, instruction: str, model):
+        """
+        Filters the DataFrame with the semantics of the user instruction.
+
+        Args:
```

Comment:
Could we please add some simple examples in the doc string?

Reply:
Sure!